synapseutils/copy_functions.py

```diff
@@ -242,27 +242,31 @@ def changeFileMetaData(
     downloadAs: str = None,
     contentType: str = None,
     forceVersion: bool = True,
+    name: str = None,
 ) -> Entity:
     """
     Change File Entity metadata like the download as name.
 
     Arguments:
         syn: A Synapse object with user's login, e.g. syn = synapseclient.login()
         entity: Synapse entity Id or object.
-        contentType: Specify content type to change the content type of a filehandle.
         downloadAs: Specify filename to change the filename of a filehandle.
+        contentType: Specify content type to change the content type of a filehandle.
         forceVersion: Indicates whether the method should increment the version of
                         the object even if nothing has changed. Defaults to True.
+        name: Specify filename to change the filename of the file.
 
     Returns:
         Synapse Entity
 
     Example: Using this function
-        Can be used to change the filename or the file content-type without downloading:
+        Can be used to change the filename, the filename when the file is downloaded, or the file content-type without downloading:
 
-        file_entity = syn.get(synid)
-        print(os.path.basename(file_entity.path))  ## prints, e.g., "my_file.txt"
-        file_entity = synapseutils.changeFileMetaData(syn, file_entity, "my_new_name_file.txt")
+            file_entity = syn.get(synid)
+            print(os.path.basename(file_entity.path))  ## prints, e.g., "my_file.txt"
+            file_entity = synapseutils.changeFileMetaData(syn, file_entity, "my_new_name_file.txt", "my_new_downloadAs_name_file.txt")
```

Comment:
This looks like it needs to be updated to reflect the change in argument order

Reply:
Great catch! I updated the order, but I also changed this to use kwargs: https://github.com/Sage-Bionetworks/synapsePythonClient/pull/1065/commits/65dd776245bfe10eeb3f46cf714ea5071ffb941e